reaper/cogs/playtester_ping.py

```diff
@@ -37,6 +37,9 @@ def getLatestDiscussion():
         response = requests.post(url, json={"query": query}, headers=headers)
         if response.status_code == 200:
             raw_data = response.json()
+        else:
+            logger.warn(f"GitHub API returned HTTP code: {response.status_code}")
```

Comment:
Changing this to an error log, than it should be good to merge :D

```suggestion
            logger.error(f"GitHub API returned HTTP code: {response.status_code}")
```